DQM/BeamMonitor/plugins/BuildFile.xml

```diff
@@ -53,3 +53,10 @@
 <library file="FakeBeamMonitor.cc" name="FakeBeamMonitor">
   <flags EDM_PLUGIN="1"/>
 </library>
+
+<library file="BeamSpotDipServer.cc" name="BeamSpotDipServer">
+  <use name="DataFormats/OnlineMetaData"/>
+  <use name="dip"/>
+  <use name="log4cplus"/>
+  <flags EDM_PLUGIN="1"/>
+</library>
```

Comment:
@francescobrivio , ah sorry one more update needed here otherwise it will fail for non-x86_64 archs. `dip` is only available for `amd64` archs , so in current situation this will break `aarch64 and ppc64le` IBs. Can you please enclose this block in 
```
<ifarchitecture name="_amd64_">

</ifarchitecture>
```

Reply:
yes please

Reply:
Addessed in https://github.com/cms-sw/cmssw/pull/35433/commits/b716e85052aff9278bf5056977f6a5c0835f534e
@sikler please copy this one as well in the backport.

Reply:
done